### PR TITLE
Corrected metadata.yml files

### PR DESCRIPTION
**Pull request by julia-pfarr**

### CODE REVIEW

File: tests/data/metadata.yml
[the remark this answers is not in the document]
Comment (julia-pfarr) [reply tests/data/metadata.yml:1]: Yes, I had it but lost it at some point. I'll put it in again

[the remark this answers is not in the document]
Comment (julia-pfarr) [reply tests/data/metadata.yml:1]: Can you see it?

[the remark this answers is not in the document]
Comment (julia-pfarr) [reply tests/data/metadata.yml:1]: Wait, it didn't save the second file (test)

File: tests/data/metadata.yml [hunk 1]
```diff
@@ -1,21 +1,23 @@
 ---
 # manual_metadata
+# Please find the BIDS specification here: https://bids-specification--1128.org.readthedocs.build/en/1128/modality-specific-files/eye-tracking.html
+# Please go through the specification and fille the fields below according to the data types given in the specification.
 
-# Required, fill according to BIDS specification:
+# REQUIRED, fill according to BIDS specification (!the converter will not run successfully if you leave this empty!):
 SampleCoordinateUnits: pixel
```
Comment (mszinte) [inline tests/data/metadata.yml:7]: I meant here the 'pixel'

Comment (mszinte) [reply tests/data/metadata.yml:7]: Also the values of size are in meter

File: tests/data/metadata.yml
[the remark this answers is not in the document]
Comment (julia-pfarr) [reply tests/data/metadata.yml:1]: I don't know, as soon as the tests are starting to run the data type entries I made in the tests/data/metadata.yml file disappear. I think it's some automatic thing from the tests and the yml configuration. 

But the important thing is that it is in the eye2bids/config/metadata.yml file as this is the file the users use as a template to fill their own stuff

[the remark this answers is not in the document]
Comment (mszinte) [reply tests/data/metadata.yml:1]: ok

**Review by mszinte — APPROVED** [review]

File: tests/data/metadata.yml (hunk 1 above)
Comment (julia-pfarr) [reply tests/data/metadata.yml:7]: > I meant here the 'pixel'

I know, like I said, I put !!str around it to make it clear that it should be a string but as soon as the CI starts here it disappears.

Comment (julia-pfarr) [reply tests/data/metadata.yml:7]: > Also the values of size are in meter

Which size? ScreenSize?

Comment (mszinte) [reply tests/data/metadata.yml:7]: Check the doc I think all size

Comment (julia-pfarr) [reply tests/data/metadata.yml:7]: Ok, the glossary says it's meter. So we need to change the example on the doc, there it is in cm. I open an issue for it.

Comment (julia-pfarr) [reply tests/data/metadata.yml:7]: Changed it in the files here, too

**Pull request merged**